l2-contracts/src/deploy-shared-bridge-on-l2-through-l1.ts

```diff
@@ -52,30 +52,8 @@ export async function publishL2NativeTokenVaultDependencyBytecodesOnL2(
   }
 }
 
-async function setL2TokenBeacon(deployer: Deployer, chainId: string, gasPrice: BigNumberish) {
-  if (deployer.verbose) {
-    console.log("Setting L2 token beacon");
-  }
-  const bridgehub = BridgehubFactory.connect(L2_BRIDGEHUB_ADDRESS, deployer.deployWallet);
-  const receipt2 = await deployer.executeUpgradeOnL2(
-    chainId,
-    L2_BRIDGEHUB_ADDRESS,
-    gasPrice,
-    bridgehub.interface.encodeFunctionData("setAddresses", [
-      L2_ASSET_ROUTER_ADDRESS,
-      ADDRESS_ONE,
-      L2_MESSAGE_ROOT_ADDRESS,
-    ]),
-    priorityTxMaxGasLimit
-  );
-  if (deployer.verbose) {
-    console.log("Set addresses in BH, upgrade hash", receipt2.transactionHash);
-  }
-}
-
```

Comment:
btw this can be clean up even more, I did it here, ( we did not merge it): 
https://github.com/matter-labs/era-contracts/pull/706
( its fine as it is ofc)